### ISSUE TITLE
Error: ENOENT: no such file or directory, open '@/assets/logo.png'

### ISSUE THREAD

**Issue by 7iomka** [IndexXuan/vue-cli-plugin-vite#36]
Hi.
I try to run [my-spa-js-app](https://github.com/IndexXuan/vue-cli-plugin-vite/tree/main/examples/my-spa-js-app)
```
yarn vite
```
dev server running ok, but url to image in css in App.vue not resolved
```
div {
  background: url(~@/assets/logo.png) 0 0 no-repeat;
}
```
and I get this error in chrome devtools
```
GET http://localhost:3000/@fs/@/assets/logo.png 404 (Not Found)
```

I read the docs, I try to run build
```
yarn vite:build
```
Then i get error in console
```
mv: no such file or directory: /Users/7iomka/Desktop/Projects/qubittech/projects/money-planet/my-spa-js-app/dist/src/*.html
[vite:css] ENOENT: no such file or directory, open '@/assets/logo.png'
file: /Users/7iomka/Desktop/Projects/qubittech/projects/money-planet/my-spa-js-app/src/App.vue?vue&type=style&index=0&lang.css
error during build:
Error: ENOENT: no such file or directory, open '@/assets/logo.png'
```

Why? How to fix that?

**Comment by IndexXuan** [comment]
will figure it soon. too busy weekdays.

**Comment by IndexXuan** [comment]
done. update vue-cli-plugin-vite to 1.3.5 in examples, it works.
It is a `css-loader` compatible issue fixed in v1.3.5.

**Comment by IndexXuan** [comment]
https://github.com/IndexXuan/vue-cli-plugin-vite/commit/e2bff0aaee628372265acc0c032706d48e0b977d
